mnq/sns-instances-notification-system/README.md

```diff
@@ -6,9 +6,11 @@ This repository contains the source code for the this tutorial: [Creating a noti
 
 This example assumes that you are familiar with:
 
-- how System Notification Service work. You can find out more in the [SNS Scaleway official documentation](https://www.scaleway.com/en/docs/serverless/messaging/quickstart/)
-- how Instances work. If needed, you can visit the [Instances Quickstart documentation](https://www.scaleway.com/en/docs/compute/instances/quickstart/).
-- how to create a SSH key. If not done already, you can follow the first 6 steps of [this walkthrough](https://www.scaleway.com/en/docs/identity-and-access-management/organizations-and-projects/how-to/create-ssh-key/#how-to-upload-the-public-ssh-key-to-the-scaleway-interface).
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Set up authentication for the Terraform provider](https://registry.terraform.io/providers/scaleway/scaleway/latest/docs#authentication)
+- An [SSH key](/identity-and-access-management/organizations-and-projects/how-to/create-ssh-key/)
+- [Set up the Scaleway CLI](/developer-tools/scaleway-cli/quickstart/)
+
```

Comment:
suggestion: The four dots seem strange with the phrase "you are familiar with".
Specifically the points:
- A Scaleway account logged into the [console](https://console.scaleway.com)
- An [SSH key](/identity-and-access-management/organizations-and-projects/how-to/create-ssh-key/) 

Maybe rephrase this sentence or revise the two dots, what do you think about it?